src/vsr/replica.zig

```diff
@@ -5448,6 +5537,8 @@ pub fn ReplicaType(
             assert(dvcs_canonical.len > 0);
 
             for (dvcs_all.constSlice()) |message| {
+                assert(message.header.op <= self.op_checkpoint_trigger());
```

Comment:
If any of the DVCs had an op belonging to the next checkpoint, then we wouldn't have started this view as the primary, we would have forfeited in favor of that other replica.